server/src/game/player/player_helper_functions.rs

```diff
@@ -158,7 +140,7 @@ impl PlayerReference{
         self.role(game).has_suspicious_aura(game) || 
         self.night_framed(game) || 
         (
-            self.doused(game) &&
+            game.arsonist_doused().doused(*self) &&
             PlayerReference::all_players(game).any(|player_ref|
                 player_ref.alive(game) && player_ref.role(game) == Role::Arsonist
             )
```

Comment:
Why not have the arsonist give players suspicious aura rather than having them each compute whether they're doused? Less coupling. Also, every player is checking whether an arsonist is alive. If the arsonist gives players suspicious aura then this only needs to be computed once.

Reply:
I don't know what your saying. This is how you give someone a suspicious aura.
This should be in a separate issue/PR anyway. I just changed this to make it not error

Reply:
It also has night_framed there. should i also change that?

I actually am going to make framed its own component anyway, but i would have this code stay just the same

game.framer_framed().framed(*self)

I dont understand what your suggesting

Reply:
`night_suspicious_aura` and then make doused and frame give sus aura

Reply:
I don't think I want to do that.

We should talk about that in person. I think this way is better